opennms-doc/guide-install/src/asciidoc/text/minion/introduction.adoc

```diff
@@ -2,16 +2,69 @@
+The _Minion_ has a configuration property for the _Location_ and will register itself on first startup on the _{opennms-product-name}_ instance.
+
+The _Provisioning System_ allows to associate _Nodes_ to a _Location_.
+_{opennms-product-name}_ will delegate monitoring requests for _Nodes_ in the specified _Locations_ to the registered _Minions_ and uses them as a proxy.
+
+Figure <<gi-install-minion-communication, Minion communication>> gives a more detailed overview about the communication between an _{opennms-product-name}_ instance and a _Minion_.
+
+[[gi-install-minion-scenario]]
+.Minion communication
+image::minion/02_minion-communication.png[]
+
+The _Minion_ needs a configuration which requires at minimum the following information:
+
+* An unique identifier for this specific _Minion_ as `id`
+* Monitoring location as a name for a network area this _Minion_ is responsible for nodes named as `location`
+* The communication endpoints for the central _{opennms-product-name}_ instance as `broker-url` and `http-url`
+
+The configuration is in a property file in `${MINION_HOME}/etc/org.opennms.minion.controller.cfg`.
+When the minimal configuration is set up the _Minion_ can be started and initially connects to the central _{opennms-product-name}_ instance and identifies itself with his unique _ID_.
+
+NOTE: If the _ID_ for a _Minion_ is not configured manually, the first start generates a unique _ID_ using `/usr/bin/uuidgen -t`.
+
+By default the _Minion_ will be automatically provisioned as a _Node_ in the _{opennms-product-name}_ instance and get automatically monitored with the _Minion-Heartbeat_ service.
+The _Minion_ sends heart beat messages to ensure it is running and functioning properly in this network area.
+
+The specific management protocol messages, e.g. _SNMP_, _ICMP_, are piped through an _Active MQ_ messaging communication channel and are executed by a _Minion_.
+Responses are forwarded to the central _{opennms-product-name}_ instance and are processed accordingly.
+
+_Minion_ can be installed on the same system as _{opennms-product-name}_ or on other systems provided that it can communicate with:
+
+* The _OpenNMS ReST Interface_, by default _TCP_ port 8980
+* The _ActiveMQ_ broker used by _{opennms-product-name}_, by default _TCP_ port 61616
+
+The following management protocols are currently supported in a _Minion_ proxy scenario:
+
+* Receive _Syslog_ messages and forward them through _Active MQ_ to a central _{opennms-product-name}_ instance
+* Receive _SNMP Traps_ and forward them through _Active MQ_ to a central _{opennms-product-name}_ instance
+* Act as a proxy for _SNMP_ performance data collections
+* Act as a proxy for _Service Monitors_ to test availability and measure response times from applications
+
+IMPORTANT: Packages are only available as RPMS for RHEL-based systems.
+
+WARNING: To avoid issues, make sure the _Minion_ and the instance of _{opennms-product-name}_ have the same version.
```

Comment:
good point, I'm unsure compatible needs to be explained, same version number not :)